cron.go

```diff
@@ -265,16 +281,28 @@ func (c *Cron) run() {
 				now = now.In(c.location)
 				c.logger.Info("wake", "now", now)
 
-				// Run every entry whose next time was less than now
-				for _, e := range c.entries {
-					if e.Next.After(now) || e.Next.IsZero() {
+				var entries []*Entry
+				index := 0
```

Comment:
I don't think you need an index var. You should be able to do this instead: `for index, entry := range`